src/Files.App/App.xaml.cs

```diff
@@ -448,19 +449,41 @@ private static void AppUnhandledException(Exception ex, bool shouldShowNotificat
 				{
 					Buttons =
 					{
-						new ToastButton("ExceptionNotificationReportButton".GetLocalizedResource(), "report")
+						new ToastButton("ExceptionNotificationReportButton".GetLocalizedResource(), Constants.GitHub.BugReportUrl)
 						{
-							ActivationType = ToastActivationType.Foreground
+							ActivationType = ToastActivationType.Protocol
 						}
 					}
-				}
+				},
+				ActivationType = ToastActivationType.Protocol
 			};
 
 			// Create the toast notification
 			var toastNotif = new ToastNotification(toastContent.GetXml());
 
 			// And send the notification
 			ToastNotificationManager.CreateToastNotifier().Show(toastNotif);
+
+			// Restart the app
+			var userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
```

Comment:
This assignation is done several times through the class, why not hoilst it on top of it in `private readonly` field?